README.md

```diff
@@ -73,7 +73,12 @@ The pot task driver supports the following parameters:
 * The pot-task-driver binary placed in the [plugin_dir][plugin_dir] directory. 
 * Installing [pot][pot-github-repo] and following the install [guide][pot-install-guide].
 * Webserver from where to serve the images. (simple file server)
+* Following lines need to be included in your rc.conf
 
+```
+nomad_user="root"
+nomad_env="PATH=/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/sbin:/bin"
```

Comment:
should we also add the VAULT_TOKEN in the env o at least mention that it is possible?

Reply:
would say leave it out as its more nomads documentation as to which ENV vars are supported